### PR TITLE
[FR] Add Tokens to PUI

### LINKED ISSUE #6500: [FR] Add Tokens to PUI
### Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find a similar feature request

### Problem statement

API Tokens are currently missing from /settings/user/security

### Suggested solution

Add API tokens to PUI /settings/user/security and managment APIs to the backend

### Describe alternatives you've considered

Using Django Admin (if you are a superuser and know about it)

### Examples of other systems

_No response_

### Do you want to develop this?

- [X] I want to develop this.

### PR DESCRIPTION
**Pull request by matmair**
This PR adds:
- an API for Token managment (listing and revoken the users own Token)
- a UI for PUI to manage the tokens (revoke them, list them)
- 
Closes #6500

### CODE REVIEW

**Comment by matmair** [comment]
@SchrodingersGat ready for review and merge

File: src/frontend/src/pages/Index/Settings/AccountSettings/SecurityContent.tsx [hunk 1]
```diff
@@ -329,3 +336,85 @@ function MfaContent() {
...
+
+  if (data.length == 0)
+    return (
+      <Alert icon={<IconAlertCircle size="1rem" />} color="green">
+        <Trans>No tokens configured</Trans>
+      </Alert>
+    );
+
+  return (
+    <Table stickyHeader striped highlightOnHover withTableBorder>
```
Comment (wolflu05) [inline src/frontend/src/pages/Index/Settings/AccountSettings/SecurityContent.tsx:394]: Q: is there a reason why you built this table yourself and didn't used the inventree table, to reduce duplicate code?

Comment (matmair) [reply src/frontend/src/pages/Index/Settings/AccountSettings/SecurityContent.tsx:394]: It took up a lot of unnecessary horizontal space and the complexity is not needed here

Comment (wolflu05) [reply src/frontend/src/pages/Index/Settings/AccountSettings/SecurityContent.tsx:394]: ok

**Comment by SchrodingersGat** [comment]
@matmair thanks for implementing, this looks good on my end :)

**Pull request merged**